mlos_bench/mlos_bench/optimizers/mlos_core_optimizer.py

```diff
@@ -117,7 +117,7 @@ def bulk_register(self,
 
         # TODO: Specify (in the config) which metrics to pass to the optimizer.
         # Issue: https://github.com/microsoft/MLOS/issues/745
-        self._opt.register(df_configs, df_scores[opt_targets].astype(float))
+        self._opt.register(configurations=df_configs, scores=df_scores[opt_targets].astype(float))
```

Comment:
```suggestion
        self._opt.register(configs=df_configs, scores=df_scores[opt_targets].astype(float))
```

Proposal to also shorten this for brevity to just `configs` everywhere.

@motus do you agree?